util/scripts/sync_tas/sync_allocation_from_tas.py

```diff
@@ -113,19 +114,25 @@ def sync(db, tas_allocs, portal_allocs):
     for tas_alloc in tas_allocs:
         if not columns:
             columns = tas_alloc.keys()
-        # tas projects_for_group gives inconsistent computeUsed
-        compare_tas_alloc = tas_alloc.copy()
-        del compare_tas_alloc['su_used']
+        compare_tas_alloc = {}
```

Comment:
I think it may be easier to reason about this by implementing a compare function:

```python
def alloc_equals(portal_alloc, tas_alloc):
    return all(tas_alloc.get(key) == portal_alloc.get(key) for key in ALLOCATION_ID_KEYS)
```

Then you can simply do

```python
for portal_alloc in portal_allocs:
    if alloc_equals(portal_alloc, tas_alloc):
        exist = True